apps/backoffice-v2/src/domains/alerts/fetchers.ts

```diff
@@ -79,3 +83,23 @@ export const fetchAlerts = async (params: {
 
   return handleZodError(error, alerts);
 };
+
+export const assignAlertsByIds = async ({
+  assigneeId,
+  alertIds,
+}: {
+  assigneeId: string | null;
+  alertIds: string[];
+}) => {
+  const [alerts, error] = await apiClient({
+    url: `${getOriginUrl(env.VITE_API_URL)}/api/v1/external/alerts/assign`,
+    method: Method.PATCH,
+    body: {
+      assigneeId,
```

Comment:
you should also pass projectid

Reply:
@alonp99 wants project ids to be handled by the server.

Reply:
@alonp99  @nitzanballerine How can we determine what is the selected project id from application perspective ?

Reply:
In the future we can use what you did with the query param - but we don't need it right away - to reduce scope we can choose the authenticated user project id by default || query params project ids